communication/slack-config/users.yaml

```diff
@@ -177,7 +178,7 @@ users:
   pohly: U91901TMF
   prajyot-parab: U02MVRCN8CX
   prietyc123: U01D4MBLM52
-  psaggu: U012EE74CU8
+  Priyankasaggu11929: U012EE74CU8
```

Comment:
`users.yaml` is supposed to map GH usernames -> slack IDs.
Not sure why `psaggu` was mapped here, might be worth auditing this later on.

Reply:
I think it was an overlook to add Priyanka's Slack username instead of GitHub username.

Reply:
Thought so too, not sure if there are other such instances in that file though.